gulpfile.js

```diff
@@ -0,0 +1,26 @@
+'use strict'; //eslint-disable-line
```

Comment:
I think we should creating a `package.json` file which adds `gulp` as a `devDependency` and this gulp task an an `npm run-script`, this way other contributors will be able to generate the `imageList.json` by doing `npm install && npm run some_task_name_here`.